README.md

```diff
@@ -152,6 +153,33 @@ function validateLength($str, $attribs, $errorCb) {
 }
 ```
 
+## Change settings page location in the menu
+
+By default, `add_plugins_page()` is used for registering the settings page. This places your settings page under the Plugins menu which you may or may not want. Should you want to change the location of the settings page, switch to `add_submenu_page()`. 
```

Comment:
Can you make `add_plugins_page` and `add_submenu_page` links to their pages in the WP documentation.